src/xdocs/config_misc.xml

```diff
@@ -1320,6 +1320,128 @@ void foo(String aFooString,
       </subsection>
     </section>
 
+    <section name="OrderedProperties">
+      <subsection name="Description" id="OrderedProperties_Description">
+        <p>Since Checkstyle 8.21</p>
```

Comment:
@thomassenger , make it 8.22.
I confident we will merge this before 8.22 release day.